torch/ao/quantization/pt2e/representation/rewrite.py

```diff
@@ -823,5 +823,6 @@ def reference_representation_rewrite(model: GraphModule) -> GraphModule:
             replacement = replacement_post_trans(replacement)
         pattern.recompile()  # type: ignore[attr-defined]
         replacement.recompile()  # type: ignore[attr-defined]
-        matches = replace_pattern(model, pattern, replacement)
+        replace_pattern(model, pattern, replacement)
```

Comment:
Does this have side effects? unsure

Reply:
I guess it does the replace inplace, so yes